salt/modules/flatpak.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+'''
+Manage flatpak packages via Salt
+
+.. versionadded:: Neon
+
+:depends: flatpak for distribution
+'''
+
+from __future__ import absolute_import, print_function, unicode_literals
+import subprocess
+import re
+
+import salt.utils.path
+
+FLATPAK_BINARY_NAME = 'flatpak'
+
+__virtualname__ = 'flatpak'
+
+
+def __virtual__():
+    if salt.utils.path.which('flatpak'):
+        return __virtualname__
+
+    return (False, 'The flatpak execution module cannot be loaded: the "flatpak" binary is not in the path.')
+
+
+def install(location, name):
+    '''
+    Install the specified flatpak package or runtime from the specified location.
+    Returns a dictionary of "result" and "output".
+    location
+        The location or remote to install from.
+    name
+        The name of the package or runtime
+    '''
+    ret = {'result': None, 'output': ""}
+
+    try:
+        ret['output'] = subprocess.check_output([FLATPAK_BINARY_NAME, 'install', location, name], stderr=subprocess.STDOUT)
```

Comment:
Is there a reason you're using `subprocess` here instead of using `__salt__['cmd.run']()` to execute these commands?
If you need the retcode, use `__salt__['cmd.run_all']()`.

This should be addressed throughout the module.

Reply:
I was copying how https://github.com/saltstack/salt/pull/51625 did it. I will try using ` __salt__['cmd.run']() `. I don't forsee that being an issue.